docs/WindowsNative.md

```diff
@@ -55,6 +58,31 @@ and you should be done!
 You only have to run `npm install` the first time;
 every time after that, you should just be able to run `npm start`.
 
+## Debugging using VSCode
+Similar to [WindowsSubsystemForLinux](WindowsSubsystemForLinux.md), the following is a `launch.json` that works for attaching to an instance of CE that was launched with `npm run-script debugger` (launches with the `--inspect` flag). 
+
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Attach to Process",
+            "port": 9229,
+            "address": "localhost",
+            "protocol": "inspector",
+            "localRoot": "${workspaceRoot}",
+            "remoteRoot": "C:\\Users\\${username}\\compiler-explorer"
+        }
+    ]
+}
+```
+
+Launch CE with `npm run-script debugger` to have node listen on port 9229. 
+
+Because you can only attach to the process, as opposed to launching the process, in order to debug startup code you need to attach while npm is starting up.  The `debugger` script also enables `debug` logging level so debug print statements can be seen during the CE startup and run.
```

Comment:
I don't think this is true. Can you rephrase to "Because this only attached to a running process, as opposed to ..."